### DISCUSSION TITLE
Where are the secret Dock settings in OS 8?

### DISCUSSION THREAD

**Discussion by danirabbit** [elementary/triage#619] [Q&A] [answered]
In OS 7.1 and before you could access a hidden dock options window with lots of extras that aren't available in System Settings. How can I access these features in OS 8?

**Comment by NutriPhil** [comment]
I did not test it, but try this in terminal:

plank --preferences

**Reply by danirabbit** [reply]
This doesn't work in OS 8. Please only provide answers that you've actually researched and tested :)

**Comment by danirabbit** [comment, marked as answer]
All of the options available for the new Dock in OS 8 are available from System Settings. **The hidden extra options that were available in Plank aren’t supported by the new Dock** which was written specifically for elementary OS and the secure session.

**Can you add the feature that I'm missing to the new dock?**
If you relied on one of these hidden options, please use the Feedback app to let us know so we can consider this feature in future updates!

**Can I theme the new dock?**
If you want to theme the new Dock, you can now do that by adding custom CSS to a CSS overrides file at `~/.config/gtk-4.0/gtk.css`.  There's a great community project here with some example themes: https://github.com/teamcons/eOSdock-themeswitcher

**Can I still use Plank instead?**
You can still install Plank (or another dock program) via Terminal and use it instead of the new Dock, but it will only work in the Classic Session. Alternative docks will not work in the Secure Session.

**Reply by danirabbit** [reply]
Sure, your local css overrides files effects the whole session. For the panel, you can see the default css supplied [here](https://github.com/elementary/wingpanel/blob/master/data/styles/Application.css) to get an idea of the selectors you'd want to override

**Reply by LeptOn30** [reply]
Hi can you tell us how to disable the new dock, so I can install plank and use it?

**Reply by typomedia** [reply]
```
sudo sed -i 's/launch-on-x=true/launch-on-x=false/g' /etc/xdg/io.elementary.desktop.wm.shell
```

**Comment by sj64** [comment]
Created ~/.config/gtk-4.0/gtk.css
added the line:
dock{opacity: alpha(@bg_color,0.0);} 
(as per themeswitcher script)
ran killall io.elementary.dock in terminal
Logged out and in - secure mode
No change to dock.

Am I missing something?

**Reply by typomedia** [reply]
```bash
mkdir -p ~/.config/gtk-4.0
echo "dock {opacity: 0;}" >>  ~/.config/gtk-4.0/gtk.css && killall io.elementary.dock
```

**Comment by Poker-Alho** [comment]
Hey @danirabbit

There's a lot of things I miss in the old dock.

You shouldn't remove features. Make it better? Yes. Remove, no. That's so Microsoft and Windows spirit.

I miss, for example, the possibility of adding the garbage bin icon to the dock; show desktop icon.

Can you add it?
